brion/plugin/morphologyHDF5.cpp

```diff
@@ -204,16 +188,14 @@ Vector4fsPtr MorphologyHDF5::readPoints(MorphologyRepairStage stage) const
         H5::DataSet dataset;
         try
         {
-            if (stage == MORPHOLOGY_UNDEFINED)
-                stage = _stage;
-            dataset = _file.openDataSet("/" + _g_root + "/" + toString(stage) +
-                                        "/" + _d_points);
+            dataset = _file.openDataSet("/" + _g_root + "/" + _stage + "/" +
+                                        _d_points);
```

Comment:
You have changed the default semantics here. Are you sure there's no tool using v2 morphologies that would be affected by this change?